### ISSUE TITLE
[Accessibility]: `--search-filter-separator-color` has too low contrast in light mode

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#34734]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34189

### Problem description

`--search-filter-separator-color`, which is used in the colon after filters, has too low contrast in light mode.

![image](https://user-images.githubusercontent.com/206864/166077893-9a5bbca6-e9d1-440d-a008-ca0d2c4de12f.png)

### Expected behavior

@sourcegraph/design to suggest a new color to replace it. It also needs to be replaced in the [Monaco theme](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/shared/src/components/MonacoEditor.tsx?L112%3A1-112%3A76=).

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by limitedmage** [comment]
@almeidapaulooliveira can you suggest an appropriate color to use here? Thanks!

**Comment by limitedmage** [comment]
Change light mode to `--oc-gray-7`
